lib/index.js

```diff
@@ -307,12 +345,17 @@ function getTokens(palette) {
 		borderColorButtonCriticalBordered: _colors.red.normal,
 		borderColorButtonCriticalBorderedHover: _colors.red.normalHover,
 		borderColorButtonCriticalBorderedActive: _colors.red.normalActive,
+		borderColorCheckboxRadio: _colors.ink.lighter,
+		borderColorCheckboxRadioError: _colors.red.normal,
+		borderColorCheckboxRadioHover: _colors.ink.light,
+		borderColorCheckboxRadioActive: _colors.ink.normal,
+		borderColorCheckboxRadioFocus: _colors.blue.normal,
 		// category:Border style
 		// description:
 		borderStyleCard: "solid",
 		borderWidthCard: "1px",
 		borderStyleInput: "solid",
-		borderWidthInput: "1px",
+		borderWidthInput: "2px",
```

Comment:
I think that there should be the old value `1px` and we should create a new token for focus border width => `borderWidthInputFocus: "2px"`

Reply:
Oh, good point!